src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -16,4 +16,5 @@ CA2011 | Reliability | Info | AvoidInfiniteRecursion, [Documentation](https://do
 CA2012 | Reliability | Hidden | UseValueTasksCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2012)
 CA2013 | Reliability | Warning | DoNotUseReferenceEqualsWithValueTypesAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2013)
 CA2014 | Reliability | Warning | DoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2014)
-CA2015 | Reliability | Warning | DoNotDefineFinalizersForTypesDerivedFromMemoryManager, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2015)
\ No newline at end of file
+CA2208 | Usage | Info | InstantiateArgumentExceptionsCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2208)
```

Comment:
Interesting - didn't the code fix for updating this file suggest creating a new `## Changed Rules` table for CA2208? It is already part of [AnalyzerReleases.Shipped.md](https://github.com/dotnet/roslyn-analyzers/blob/62e23a714e3609c6bb9daac1dfc9fd3cc6a152e2/src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md), so I would have expected so. I will investigate why the code fix didn't do so.

Reply:
no idea, i got build error error saying something like "existing CA2208 updated but not listed in Unshipped.md" don't remember exact wording

Reply:
That's also funny because the ordering was broken. The code-fix was doing a good job at keeping the order...

Reply:
Alright, so I manually edited the descriptors for `InstantiateArgumentExceptionsCorrectlyAnalyzer` to use `RuleLevel.IdeSuggestions`, and the code fix does offer adding an entry for changed rule:

![ReleaseTrackingCodeFix](https://user-images.githubusercontent.com/10605811/80042455-dec9c900-84b3-11ea-99ea-284613b9b29c.png)

Kindly revert the manual update to this file, and apply the code fix to do this correctly.

Reply:
Oh sorry, didn't noticed there were analyzer and fixer, its worked as needed, thanks!